src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,19 @@
-export const TodoInfo = () => {};
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const completed = `TodoInfo${
+    todo.completed ? ' TodoInfo--completed' : ''}`;
```

Comment:
use destructuring for the todo to improve the code readability
const { completed, id, ... } = todo;
```suggestion
  const completed = `TodoInfo${completed ? ' TodoInfo--completed' : ''}`;
```